src/Database/Table/Selection.php

```diff
@@ -291,7 +291,7 @@ public function wherePrimary(mixed $key): static
 
 	/**
 	 * Adds where condition, more calls appends with AND.
-	 * @param  string|string[]  $condition  possibly containing ?
+	 * @param  string|array<int|string, mixed>  $condition  possibly containing ?
```

Comment:
`array<int|string, mixed>` is simply `array`

Reply:
So should we use just array? :) or should we list all types instead of mixed?

Just from one of my projects there are: int|string|bool|array of whatever |DateTime|ActiveRow|Selection and there can be many others I think

cc @ondrejmirtes

Reply:
Klidně tam dej array

Reply:
hotovo ;)